app/views/items/index.html.erb

```diff
@@ -0,0 +1,55 @@
+<center>
+  <section id="item-title">
+    <% if @merchant %>
+    <h1><a href="/merchants/<%=@merchant.id%>"><%= @merchant.name %> Items</h1>
+    <% else %>
+      <h1>All Items</h1>
+    <% end %>
+  </section>
+
+<section class="boxes-container">
+  <% @items.each do |item| %>
+  <% if @merchant %>
+    <% if item.active? %>
+      <section class="single-box">
+        <div id="item-pic">
+          <%= link_to "/items/#{item.id}" do %>
+            <img src="<%=item.image %>" id="single-pic">
+          <% end %>
+        </div>
+        <div class="">
+          <p><%= link_to "#{item.name}", "/items/#{item.id}", method: :get %></p>
+          <p>description:<%= item.description %></p>
+          <p>price:<%= number_to_currency(item.price) %></p>
+          <p>inventory:<%= item.inventory %></p>
+          <p>active:<%= item.active? %></p>
+          <p><%= link_to "#{Merchant.find(item.merchant_id).name}", "/merchants/#{item.merchant_id}", method: :get %></p>
```

Comment:
```html.erb
<%= link_to "#{Merchant.find(item.merchant_id).name}", "/merchants/#{item.merchant_id}", method: :get %>
```
You have some logic in this link above. Since your items have a merchant id, you can call `item.merchant` and you will get returned to you that items merchant. Since you just want that link to be the name of the merchant, you can do: 
```html.erb
<%= link_to item.merchant.name, "/merchants/#{item.merchant_id}", method: :get %>
```